### PR TITLE
[not ready] Food and plants.

### PR DESCRIPTION
**Pull request by DaddyIssues98**
## About The Pull Request

This should add 2 new plants, new recipes, food and 2 new figures.

Thing to do:

- [ ]  Finish the heart-fruit plant
- [ ]  Finish the strawberry plant
- [ ]  (maybe) adds mutations to the plants
- [ ]  Add atleast 4 new recipes for heart-fruit
- [ ] Add atleast 4 new recipes for strawberry
- [x] Adds the box with the figures to the chaplain wardrobes
- [ ]  Finish the sprites
- [ ] do the code

porting this also https://github.com/fulpstation/fulpstation/pull/123 (except the sausage bread sprite)

## Why It's Good For The Game

This will add new recipes for the chefs, new plants for the botanist and the chaplain will get  a box (in his wardrobe) with figures that are  required to do the wedding cake, so he have more RP opportunites

## Changelog
:cl:
 help
/:cl:

### CODE REVIEW

File: code/game/objects/items/food/cake.dm [hunk 1]
```diff
@@ -465,3 +465,75 @@
 	food_reagents = list(/datum/reagent/consumable/nutriment = 3, /datum/reagent/consumable/nutriment/vitamin = 1)
 	tastes = list("cake" = 4, "butter" = 2, "cream" = 1)
 	foodtypes = GRAIN | DAIRY | SUGAR
+
+/obj/item/food/cake/heartfruit
+	name = "heart-fruit cake"
+	desc = "a cake full of lov- why is beating!?"
```
Comment (BootlegBow) [inline code/game/objects/items/food/cake.dm:471]: why is beating

File: code/game/objects/items/food/cake.dm [hunk 2]
```diff
@@ -465,3 +465,75 @@
...
+	desc = "A wedding cake."
+	icon_state = "weddingcake1"
+	bite_consumption = 7
+	max_volume = 100
+	food_reagents = list(/datum/reagent/consumable/nutriment = 35, /datum/reagent/consumable/nutriment/vitamin = 3, /datum/reagent/consumable/sugar = 15)
+	tastes = list("sweetness" = 2, "cake" = 5, "divorce" = 1)
+	w_class = WEIGHT_CLASS_BULKY
+	foodtypes = GRAIN | DAIRY | SUGAR
+
+//male + male  cake
```
Comment (BootlegBow) [inline code/game/objects/items/food/cake.dm:500]: gay

File: code/game/objects/items/food/misc.dm [hunk 3]
```diff
@@ -702,6 +713,17 @@
 	foodtypes = JUNKFOOD | SUGAR
 	w_class = WEIGHT_CLASS_TINY
 
+
+/obj/item/food/strawberrychocolate
+	name = "Strawberry covert in chocolate"
```
Comment (BootlegBow) [inline code/game/objects/items/food/misc.dm:718]: the strawberry is covert, i repeat we do not know where the strawberry is

File: code/game/objects/items/food/misc.dm [hunk 4]
```diff
@@ -702,6 +713,17 @@
...
+	desc = "desc"
+	icon_state = "strawberrychcolate"
```
Comment (BootlegBow) [inline code/game/objects/items/food/misc.dm:720]: chcolate

File: code/game/objects/items/food/pastries.dm [hunk 5]
```diff
@@ -549,6 +549,16 @@
 	foodtypes = GRAIN | SUGAR
 	w_class = WEIGHT_CLASS_SMALL
 
+/obj/item/food/heartfruitcookie
+	name = "Heart-fruit cookie"
+	desc = ""
```
Comment (BootlegBow) [inline code/game/objects/items/food/pastries.dm:554]: uh

File: code/game/objects/items/food/pie.dm [hunk 6]
```diff
@@ -266,3 +266,22 @@
 	icon_state = "baklavaslice"
 	tastes = list("nuts" = 1, "pie" = 1)
 	foodtypes = GRAIN
+
+/obj/item/food/pie/lemon
+	name = "lemon pie"
+	desc = "desc."
```
Comment (BootlegBow) [inline code/game/objects/items/food/pie.dm:272]: what

File: code/modules/hydroponics/grown/heartfruit.dm [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+	growing_icon = 'icons/obj/hydroponics/growing_fruits.dmi'
+	icon_grow = "heartfruit-grow"
+	icon_dead = "heartfruit-dead"
+	genes = list(/datum/plant_gene/trait/repeated_harvest)
+	reagents_add = list(/datum/reagent/consumable/nutriment/vitamin = 0.03, /datum/reagent/consumable/nutriment = 0.1)
+
+/obj/item/food/grown/heartfruit
+	seed = /obj/item/seeds/heartfruit
+	name = "heart-fruit"
+	desc = "a fruit that have a very similar shape of a heart."
```
Comment (BootlegBow) [inline code/modules/hydroponics/grown/heartfruit.dm:21]: should be has, not have

**Review by BootlegBow — COMMENTED** [review]
it looks pretty good, just a few grammatical errors

**Comment by DaddyIssues98** [comment]
going to leave this for a time

**Pull request closed without merging**